exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "ac19449e-f9a1-4b35-b73b-e35771031f20",
+   "metadata": {},
+   "source": [
+    "## Customizing Script-Language Containers\n",
+    "Sometimes you need very specific dependencies or versions of dependencies in the Exasol UDFs. In such case you can customize a Script-Language Container.\n",
+    "### Flavor Definition\n",
+    "![image.png](attachment:47f643bc-704e-4b00-8405-917b5fdae990.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58895979-99f7-4fad-ab0f-9cfa020b3750",
+   "metadata": {},
+   "source": [
+    "For customizing a flavor usually the flavor_customization build step is most important. It contains everything you need to add dependencies. The remaining build steps should be only changed with care, but sometimes some dependencies are defined in other build steps because the script client depends on them. \n",
```

Comment:
Please check, if "flavor" is used as a synonym for "container". If so, then please only use one term. 
If you still want to use the word "flavor" then maybe the following phrase could help? 
> For creating a custom container by based on a default flavor, ...